daemon/config_windows.go

```diff
@@ -0,0 +1,26 @@
+package daemon
+
+import (
+	"os"
+
+	flag "github.com/docker/docker/pkg/mflag"
+)
+
+// PlatformSpecific defines the configuration of a docker daemon
+// that is specific to a particular platform.
+type PlatformSpecific struct {
```

Comment:
@jhowardmsft I think instead of having this `PlatformSpecific struct`, we should have `Config struct` in _windows & _linux files and then a `CommonConfig` in `config.go` that we embed here like:

```go
type Config struct {
    CommonConfig
    f platformSpecificField
}
```

that sounds more reasonable to me as it doesn't introduce another level of nesting (i.e. `cfg.PlatformSpecific.SocketGroup`) and solves the vague name problem (`type PlatformSpecific`)